pygmt/src/wiggle.py

```diff
@@ -65,12 +72,12 @@ def wiggle(self, data=None, x=None, y=None, z=None, **kwargs):
         **+w**\ *length*\ [**+j**\ *justify*]\ [**+al**\|\ **r**]\
         [**+o**\ *dx*\ [/*dy*]][**+l**\ [*label*]].
         Defines the reference point on the map for the vertical scale bar.
-    color : str
+    fill : str
         Set fill shade, color or pattern for positive and/or negative wiggles
         [Default is no fill]. Optionally, append **+p** to fill positive areas
         (this is the default behavior). Append **+n** to fill negative areas.
         Append **+n+p** to fill both positive and negative areas with the same
-        fill. **Note**: You will need to repeat the color parameter to select
+        fill. **Note**: You will need to repeat the fill parameter to select
```

Comment:
> Append **+n+p** to fill both positive and negative areas with the same fill.

I haven't work on the `wiggle` module because I'm wondering if we should add parameters `fill_positive` and `fill_negative` to set the fills of the two parts separately.

Reply:
Good idea in my opinion since it's more pythonic.

Reply:
Ping @GenericMappingTools/pygmt-maintainers for comments on the parameter changes before @michaelgrund start to work on it.

Reply:
Best to do the `fill_positive` and `fill_negative` in a separate PR, if others agree too.

One question though, is there a way to make things backward compatible? I.e. have `color`, `fill_positive` and `fill_negative` be valid in PyGMT v0.8.0?

Reply:
> Best to do the `fill_positive` and `fill_negative` in a separate PR, if others agree too.

I think a separate PR is not necessary. If we decide to use `fill_positive` and `fill_negative`. It means we're deprecating the `color` parameter in v0.8.0 (removed in v0.12.0) and add two new parameters `fill_positive` and `fill_negative` in v0.8.0.

> One question though, is there a way to make things backward compatible? I.e. have `color`, `fill_positive` and `fill_negative` be valid in PyGMT v0.8.0?

See above. So the only thing we need to is checking if both `color` and `fill_positive`/`fill_negative` are specified.

Reply:
Ah ok, that works too. The implementation might be a little trickier, but let's see.

Reply:
I agree this is a good suggestion @seisman. Two separate parameters are more readable and Pythonic. Furthermore, the argument for `color` (or `fill`) is currently a bit unintuitive, as `+n` and `+p` have to be added _after_  the desired fill color.

Reply:
``fill_positive`` and ``fill_negative`` are implemented, tests and gallery examples are updated/adjusted. Needs a new round of review now @GenericMappingTools/pygmt-maintainers.